columnflow/tasks/framework/base.py

```diff
@@ -448,6 +448,35 @@ def get_array_function_kwargs(cls, task=None, **params):
 
         return kwargs
 
+    @classmethod
+    def resolve_config_parameter(
+        cls,
+        config_inst: od.Config,
+        param: tuple[str] | None = None,
+        param_name: str | None = None,
+    ) -> tuple[str]:
+        """
+        Resolves tuple of inputs *param* by using groups defined in the *config_inst*
+        that map a string to a tuple of strings. If *param* is empty, the default
+        defined in the config is returned.
+        """
+        if param and param != law.NO_STR:
+            param = law.util.make_list(param)
+
+            # expand to param group
+            param_group = config_inst.x(f"{param_name}_groups", {})
+            if param_group:
+                outp = []
+                for val in param:
+                    outp.extend(param_group.get(val, [val]))
+            return tuple(outp)
+
+        # use default if available and no param is set
+        if outp := config_inst.x(f"default_{param_name}", None):
+            return outp if isinstance(outp, tuple) else (outp,)
+
+        return ()
```

Comment:
I thought about it again, and maybe we should make the cases which are resolved a bit more expressive. Suggestion:

```python
# expand default when param is empty
if not param or param == law.NO_STR:
    param = config_inst.x(f"default_{param_name}", None)
    if not param:
        return ()

# expand to groups
if param_groups := config_inst.x(f"{param_name}_groups", {}):
    values = []
    for val in law.util.make_list(param):
        values.extend(param_group.get(val, [val]))
    return tuple(values)

# return the param as is
return law.util.make_tuple(param)
```